.github/workflows/release-pr.yml

```diff
@@ -9,6 +9,6 @@ on:
 
 jobs:
    release-pr:
-      uses: OliverMKing/javascript-release-workflow/.github/workflows/release-pr.yml@main
+      uses: Azure/action-release-workflows/.github/workflows/release_js_project.yaml@a705b2ab6a3ee889f2b0d925ad0bd2f9eb733ce6
```

Comment:
is there no tag for this hash? feel like a human-readable version would be cleaner/easier to maintain in the future

Reply:
yeah we can add tags in the future, but right now we've been updating the hash each time so the tags would just be aliases per hash

Reply:
hash is the best practice. tags are not as secure. We should continue to use sha for all action pinning